src/realm/node.cpp

```diff
@@ -110,7 +103,7 @@ void Node::alloc(size_t init_size, size_t new_width)
         // Update this accessor and its ancestors
         m_ref = mem_ref.get_ref();
         m_data = get_data_from_header(header);
-        // FIXME: Trouble when this one throws. We will then leave
+        // Trouble when this one throws. We will then leave
```

Comment:
Are you removing the FIXME because we can't do anything about it?
Could anything be persisted after this?